olive/passes/onnx/pre_post_processing.py

```diff
@@ -0,0 +1,84 @@
+from pathlib import Path
+from typing import Any, Callable, Dict, List
+
+from onnxruntime import __version__ as OrtVersion
+
+from olive.model import ONNXModel
+from olive.passes import Pass
+from olive.passes.pass_config import PassConfigParam
+
+
+class PrePostProcessing(Pass):
+    """
+    Add Pre/Post nodes to the input model
+    """
+
+    @staticmethod
+    def _default_config() -> Dict[str, Dict[str, Any]]:
+        return {
+            "pre": PassConfigParam(
+                type_=List[str],
+                default_value=None,
+                description=("List of pre-processing commands to add."),
+            ),
+            "post": PassConfigParam(
+                type_=List[str],
+                default_value=None,
+                description=("List of post-processing commands to add."),
+            ),
+            "tool_command": PassConfigParam(
+                type_=str,
+                default_value=None,
+                description=("Composited tool commands to invoke."),
+            ),
+            "tool_command_args": PassConfigParam(
+                type_=Dict[str, Any], default_value=None, description="Arguments to pass to tool command."
+            ),
+            "target_opset": PassConfigParam(
+                type_=int, default_value=16, description="The version of the default (ai.onnx) opset to target."
+            ),
+        }
+
+    def _run_for_config(self, model: ONNXModel, config: Dict[str, Any], output_model_path: str) -> ONNXModel:
+        tool_command = config.get("tool_command")
+        if tool_command:
+            if tool_command == "whisper":
+                from whisper import add_pre_post_processing_to_model as add_ppp
```

Comment:
the import needs to be an absolute path since it will be used in the package. `from olive.passes.onnx.whisper import ....`

Reply:
done